library/src/animationManager.cxx

```diff
@@ -204,6 +204,17 @@ void animationManager::Tick()
 {
   if (this->Playing)
   {
+    std::chrono::steady_clock::time_point currentTick = std::chrono::steady_clock::now();
+    if (!vtksys::SystemTools::HasEnv("CTEST_F3D_PREVENT_SKIP_FRAME") &&
```

Comment:
I remember that querying an environment variable is actually quite expensive.  
Can we move it to `animationManager` constructor and save it to a class member?

Reply:
good point.